virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc

```diff
@@ -0,0 +1,29 @@
+[id="virt-creating-data-volumes"]
+= Creating data volumes using profiles
+include::modules/virt-document-attributes.adoc[]
+:context: virt-creating-data-volumes
+
+toc::[]
+
+When you create a data volume, the Containerized Data Interface (CDI) creates a provisioned volume claim (PVC) and loads the PVC with your data. You can create a data volume as either a standalone resource or by using a virtual machine specification in a template.
```

Comment:
PVC = persistent volume claim
replace loads with populates
You can create a data volume as either a standalone resource or by using a dataVolumeTemplate in a Virtual Machine specification

Reply:
OMG, I could kill myself for making this mistake again (provisioned!!) Sorry.

I'll fight for the "populates" --- peer reviewers may push back on it as not minimal enough.

Here's the revision:

_When you create a data volume, the Containerized Data Interface (CDI) creates a persistent volume claim (PVC) and populates the PVC with your data. You can create a data volume as either a standalone resource or by using a `dataVolumeTemplate` in a Virtual Machine specification_